### PR TITLE
[3.4] Update logic for controller action arguments

### PR DESCRIPTION
**Pull request by weaverryan**
I haven't used binding yet, but I believe this is correct. @yceruto does this look good to you? You have more experience with bind than I do :).

### CODE REVIEW

File: service_container.rst [hunk 1]
```diff
@@ -690,6 +690,13 @@ service whose id is ``monolog.logger.request``.
     the *service* whose id is ``monolog.logger.request``, and not just the *string*
     ``monolog.logger.request``.
 
+.. _services-binding:
+
+Binding Arguments by Name or Type
+---------------------------------
+
+You can also use the ``bind`` keyword to bind specific arguments by name or type.
+
```
Comment (weaverryan) [inline service_container.rst]: This is a poor bootstrap of the `bind` section, just so I can link to it. i've commented in the issue about that feature (#8276) to note that this was created.

**Review by javiereguiluz — APPROVED** [review]
This looks OK to me ... but it's missing one of the most important reasons to use `bind`: define the scalar arguments once instead of repeating them in several services. It's explained in the first part of this blog post: https://symfony.com/blog/new-in-symfony-3-4-local-service-binding

**Review by yceruto — APPROVED** [review]
Yeah, it look great!

**Comment by weaverryan** [comment]
@javiereguiluz It was tricky when I talked about bind for the controller args... as I could have shown a bind on *just* that one service (a bit more clear... but kinda less useful) or apply a bind to `_defaults`, which is *awesome*, but might be a little less clear. I'm not sure what the right thing is.

But, we will of course talk about that a lot more in the `service_container.rst` chapter... which I hope we will turn into a "getting started" guide ;)

**Pull request merged**